firmware/user/modes/mode_tunernome.c

```diff
@@ -505,7 +585,7 @@ void ICACHE_FLASH_ATTR tunernomeButtonCallback( uint8_t state __attribute__((unu
                 if(button == UP || button == DOWN)
                 {
                     led_t leds[NUM_STRINGS] = {{0}};
-                    if (tunernome->currentMode == 0)
+                    if (tunernome->curTunerMode == 0)
```

Comment:
I realize it was using numerical comparisons before, but try to compare `curTunerMode` to the enum types instead, like here use `GUITAR_TUNER` instead of `0`.

Reply:
This is set up this way to wrap between the lowest mode (always 0, not necessarily always GUITAR_TUNER) and the highest mode, when clicking up/down to cycle them

Reply:
Oh, that makes sense. It's a knee-jerk reaction whenever I see an enum being compared to a number.

Reply:
Entirely fair, I had the same knee-jerk reaction when I saw your comment here, but then I went and found a spot where it was being done and thought on it some.